directives/thebe.py

```diff
@@ -149,9 +170,14 @@ def update_thebe_context(app, doctree, docname):
             repo: "{org}/{repo}",
             ref: "{branch}",
         }},
-        codeMirrorConfig: {{
-            theme: "{codemirror_theme}",
-            mode: "{cm_language}"
+        codeMirrorconfig: {{
+            theme: '{codemirror_theme}',
+            mode: '{codemirror_mode}',
+            lineNumbers: {codemirror_line_numbers},
+            electricChars: {codemirror_electric_chars},
+            indentUnit: {codemirror_indent_unit},
+            indentWithTabs: {codemirror_indent_with_tabs},
```

Comment:
Python boolean values are inserted into JavaScript here. I don't think this works because the boolean literals are different: Python `True, False` versus `true, false` in JavaScript. JavaScript crashes when it tries to execute a literal `True`: Uncaught ReferenceError: True is not defined.

Boolean variables:
```
codemirror_indent_with_tabs = True
codemirror_electric_chars = True
codemirror_line_numbers = True
```

Reply:
I guess you were testing this when conf.py had `codemirror-config` with `lineNumbers = "true"` (string `"true"`).

Reply:
Yes, i was testing with codemirror-config with correct strings. Now, I changed these to strings.